kml/types.hpp

```diff
@@ -324,6 +332,8 @@ struct TrackData
   Properties m_properties;
 };
 
+// Use this copy of structure in files with old versions (types_v<n>.hpp)
+// when new version will be implemented.
```

Comment:
Такой комментарий не будет точнее:
This structure is used in FileDataV6 because its binary format is the same as in kmb version 6. Make a copy of this structure in a file types_v<n>.hpp in case of any changes of its binary format.

Reply:
Я не против заменить коммент, но не хотелось бы в нем писать про конкретную версию, т.к. если в V7 структура CategoryData не будет изменена, то коммент устареет. При этом структуры ColorData и TrackLayer отсутствуют и в файле v3 и в файле v6. Хотелось бы что-то общее чтоб при попытке изменить структуру программист заметил коммент и обратил внимание на файлы v3 и v6. Готов к твоим предложениям по изменению коммента.

Reply:
Моя задумка была в том, что при добавлении V7 этот комментарий будет дополнен, что, скажем, и FileDataV7 использует эту структуру тоже. То есть актуальность комментариев будет поддерживаться так же, как у нас обновляются комментарии внутри kml::binary::DeserializerKml::Deserialize. Я больше опасаюсь, что слишком общий комментарий может быть трактован неверно. Структуры ColorData и TrackLayer кажутся уже сложившимися, чтобы меняться от версии к версии, поэтому не дублируются, как не дублируется и m2::PointD, например.
enum-ы цветов и типов, например, тоже будут только дополняться, их бинарный формат не меняется, меняется только диапазон значений, поэтому они не дублируются, а просто повышается версия kmb.

Кстати, можно попробовать вместе с комментарием сделать в файле types.hpp
```
using CategoryDataV6 = CategoryData;
using BookmarkDataV6 = BookmarkData;
```
а в файле types_v6.hpp
```
struct FileDataV6
{
...
  // Category's data.
  CategoryDataV6 m_categoryData;
  // Bookmarks collection.
  std::vector<BookmarkDataV6> m_bookmarksData;
  // Tracks collection.
  std::vector<TrackDataV6> m_tracksData;
...
};
```

Reply:
окей, добавил 
```
// This structure is used in FileDataV6 because 
// its binary format is the same as in kmb version 6. 
// Make a copy of this structure in a file types_v<n>.hpp 
// in case of any changes of its binary format.
```
для CategoryData и BookmarkData

Про алиасы думал, но добавлять не хотелось бы, чтобы не было ощущения что у нас где-то есть копии как в v3